### DISCUSSION TITLE
directory copy

### DISCUSSION THREAD

**Discussion by sj6219** [rustdesk/rustdesk#933] [General]
The directory does not seem to work among the file copy and paste functions. 
I think it would be useful to implement a directory copy.

**Comment by rustdesk** [comment]
@fufesou

**Comment by fufesou** [comment]
> The directory does not seem to work among the file copy and paste functions. I think it would be useful to implement a directory copy.

What's your OS on both side? Directory copy woks only on Windows both sides.

**Comment by sj6219** [comment]
I was mistaken. Now that I've tried it again, it's working well.
